### ISSUE TITLE
WCAG 4.1.2: Ensure buttons have discernible text (fluent-button[_bl_9e1f718c-5574-4e90-b63a-de5c1aba3a0b=""],.icon-only[value=""])

### ISSUE THREAD

**Issue by adamint** [dotnet/aspire#8857]
#### Issue
`Ensure buttons have discernible text` ([button-name](https://accessibilityinsights.io/info-examples/web/button-name)) [all hidden buttons + the collapse/expand button on resources + trace details]
#### Target application
[TestShop resources](https://localhost:15887/)
#### Element path
fluent-button[_bl_9e1f718c-5574-4e90-b63a-de5c1aba3a0b=""],.icon-only[value=""]
#### Snippet
`<button class="control icon-only" part="control" type="button" value=""><!----><span part="start"><slot name="start"></slot></span><span class="content" part="content"><slot></slot></span><!----><span part="end"><slot name="end"></slot></span></button>`
#### How to fix
    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element does not have an implicit (wrapped) <label>
      Element does not have an explicit
